Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -143,10 +143,6 @@
 
     <ECEntityClass typeName="DrivewayMidVertex" modifier="Sealed" displayLabel="Driveway Mid Vertex" description="Vertex of a DrivewayPath that is attached to two DrivewayPathEdge instances.">
         <BaseClass>PathMidVertex</BaseClass>
-
-        <ECProperty propertyName="SideTransitionOffset" typeName="double" kindOfQuantity="rru:LENGTH" category="DrivewayMidVertex_Transition" displayLabel="Transition Offset" description="The transition offset distance." />
```

Comment:
You can't delete properties - this is a breaking change, see https://www.itwinjs.org/bis/guide/schema-evolution/schema-versioning-and-generations/#minor-schema-upgrades

Reply:
We probably want to hide these properties and mark them as deprecated.